src/pages/billing/List.js

```diff
@@ -40,28 +74,28 @@ const NewBillingProjectModal = ajaxCaller(class NewBillingProjectModal extends C
     this.state = {
       billingProjectName: '',
       billingProjectNameTouched: false,
-      billingAccount: '',
-      existing: []
-    }
-  }
-
-  async loadBillingAccounts() {
-    const { ajax: { Billing } } = this.props
-    try {
-      const billingAccounts = await Billing.listAccounts()
-      this.setState({ billingAccounts })
-    } catch (error) {
-      reportError('Error loading billing accounts', error)
+      existing: [],
+      isBusy: false
```

Comment:
We should get in the habit of declaring all of the state keys we plan to use in the component in the constructor (I'm very bad at this myself...), so you should add `chosenBillingAccount`, `billingAccounts`, etc.

Reply:
I agree with `chosenBillingAccount` but I'm going to intentionally leave out `billingAccounts`.

If I define the billing accounts in the constructor, I get this:
<img width="1029" alt="Screen Shot 2019-04-03 at 5 37 47 PM" src="https://user-images.githubusercontent.com/2395211/55514945-57ef4780-5637-11e9-9905-32a11bc90932.png">

If I don't define them, it looks like this:
<img width="786" alt="Screen Shot 2019-04-03 at 5 37 06 PM" src="https://user-images.githubusercontent.com/2395211/55514941-545bc080-5637-11e9-9bdd-841ad2bf4163.png">

When you open the modal, if a user does not have billing accounts then it will show a specific message. If a user's billing accounts are still loading, then we show a spinner. I don't want to show that message while the billing accounts are still loading.